docs/install/offline.md

```diff
@@ -194,6 +194,15 @@ Coder also provides offline documentation in case you want to host it on your ow
 2. Download the documentation files from the "Assets" section. It is named as `coder_docs_<version>.tgz`.
 3. Extract the file and move its contents to your server folder.
 4. If you are using NodeJS, you can execute the following command: `cd docs && npx http-server .`
-5. Set the [CODER_DOCS_URL](../cli/server#--docs-url) environment variable to use the URL of your hosted docs. This way, the Coder UI will reference the documentation from your specified URL.
+5. Set the [CODER_DOCS_URL](../cli/server.md#--docs-url) environment variable to use the URL of your hosted docs. This way, the Coder UI will reference the documentation from your specified URL.
```

Comment:
this resolves https://github.com/coder/coder/actions/runs/5783167337/job/15671433381#step:4:682